packages/workers-shared/package.json

```diff
@@ -31,6 +31,8 @@
 		"check:type": "pnpm run check:type:tests && tsc",
 		"check:type:tests": "tsc -p ./asset-worker/tests/tsconfig.json",
 		"clean": "rimraf dist",
+		"deploy": "pnpm run deploy:router-worker && pnpm run deploy:asset-worker",
+		"deploy:router-worker": "CLOUDFLARE_API_TOKEN=$WORKERS_DEPLOY_AND_CONFIG_CLOUDFLARE_API_TOKEN wrangler versions upload --experimental-versions -c router-worker/wrangler.toml",
```

Comment:
I thought @GregBrimble said that we needed to provide some additional metadata for this deployment?

Reply:
correct. added in https://github.com/cloudflare/workers-sdk/pull/6543/commits/56a3866b7291a0f932718f7b9b55186c4d10e8be